docs/contents/user-docs/irsa.md

```diff
@@ -112,68 +113,43 @@ ACK_K8S_RELEASE_NAME=ack-$AWS_SERVICE_NAME-controller
 kubectl create namespace "$ACK_K8S_NAMESPACE"
 cd "$ACK_CONTROLLER_HELM_CHARTS_DIR"
 
-# dry run and view the resultant output
+# Dry run and view the output
 helm install --debug --dry-run --namespace "$ACK_K8S_NAMESPACE" "$ACK_K8S_RELEASE_NAME" .
-# install on cluster
+# Install the ACK service controller on your cluster
 helm install --namespace "$ACK_K8S_NAMESPACE" "$ACK_K8S_RELEASE_NAME" .
 ```
 
-Verify that the service account has been created on cluster and that its annotation include IAM Role
- (created during Step#2 above) arn:
+Verify that the service account has been created on your cluster and that its annotations include the IAM role for your ACK service controller.
 ```
 kubectl describe serviceaccount/$ACK_K8S_SERVICE_ACCOUNT_NAME -n $ACK_K8S_NAMESPACE
 ```
 
-#### 3(b) - If Helm charts have already been installed on cluster without modifying `values.yaml`
-
-For example, if installation was done as:
-```
-AWS_SERVICE_NAME='elasticache'
-ACK_K8S_NAMESPACE=ack-system
-ACK_K8S_RELEASE_NAME=ack-$AWS_SERVICE_NAME-controller
-helm install --namespace $ACK_K8S_NAMESPACE ack-$AWS_SERVICE_NAME-controller $ACK_K8S_RELEASE_NAME
-```
-Then service account would already exist on the cluster; however its association with IAM Role would be pending.
-Verify it using:
-```
-kubectl describe serviceaccount/$ACK_K8S_SERVICE_ACCOUNT_NAME -n $ACK_K8S_NAMESPACE
-```
-Observe that the arn of IAM Role (created during Step#2 above) is not set as annotation for the service account.
+For detailed instructions, refer to Amazon EKS documentation on how to [associate an IAM role to a service account ][5].
 
-To associate an IAM role to service account:
-```
-# annotate service account with service role arn.
-ISRA_ROLE_ARN=<role arn>
-kubectl annotate serviceaccount -n $ACK_K8S_NAMESPACE $ACK_K8S_SERVICE_ACCOUNT_NAME $ISRA_ROLE_ARN
-```
+## Step 4: Verify successful setup
 
-Update aws region to use in the controller, if not done already:
-```
-# update desired AWS region. example: us-east-1
-AWS_REGION=<aws region id>
-kubectl -n $ACK_K8S_NAMESPACE set env deployment/$ACK_K8S_RELEASE_NAME \
-    AWS_REGION="$AWS_ACCOUNT_ID"
-```
+When AWS clients or SDKs connect to an AWS API, they detect an [AssumeRoleWithWebIdentity][2] security token to assume the IAM role. 
 
-### Verify
-Describe one of the pods and verify that the `AWS_WEB_IDENTITY_TOKEN_FILE` and `AWS_ROLE_ARN` environment variables exist.
+Verify that the `AWS_WEB_IDENTITY_TOKEN_FILE` and `AWS_ROLE_ARN` environment variables exist for your Kubernetes pod using the following commands:
 ```
 kubectl get pods -A
-kubectl exec -n kube-system aws-node-<9rgzw> env | grep AWS
+kubectl exec -n <NAMESPACE> <NAME> env | grep AWS
```

Comment:
You can use `$ACK_K8S_NAMESPACE` here. Also not sure why it's grepping for `AWS` - should really be grepping for `ack`. This is probably just an artifact of an older version